media-converter/src/videoconv.rs

```diff
@@ -431,12 +431,12 @@ impl ElementImpl for VideoConv {
 }
 
 struct StreamSerializer<'a> {
-    stream: &'a Vec<u128>,
+    stream: &'a [u128],
```

Comment:
Borrowing on a Vec is just a slice, no reason to immediately not use a array then